polaris/dataset/_dataset.py

```diff
@@ -516,8 +516,8 @@ def __getitem__(self, item):
 
     def _repr_dict_(self) -> dict:
         """Utility function for pretty-printing to the command line and jupyter notebooks"""
-        repr_dict = self.model_dump()
-        repr_dict.pop("table")
+        repr_dict = self.model_dump(exclude={"table"})
```

Comment:
**Suggestion:** Let's exclude the checksum manifest as well. This doesn't hold information that the user needs and can get quite lengthy.

```suggestion
        repr_dict = self.model_dump(exclude={"table", "zarr_md5sum_manifest"})
```